Source/Scene/PointCloudEyeDomeLighting.js

```diff
@@ -0,0 +1,400 @@
+
+        var blendFS = addConstants(PointCloudEyeDomeLightingShader, 'epsilon8', CesiumMath.EPSILON8);
+
+        var blendUniformMap = {
+            u_pointCloud_colorTexture : function() {
+                return processor._colorTexture;
+            },
+            u_pointCloud_ecAndLogDepthTexture : function() {
+                return processor._ecAndLogDepthTexture;
+            },
+            u_distancesAndEdlStrength : function() {
+                distancesAndEdlStrengthScratch.x = processor._radius / context.drawingBufferWidth;
+                distancesAndEdlStrengthScratch.y = processor._radius / context.drawingBufferHeight;
+                distancesAndEdlStrengthScratch.z = processor._strength;
+                return distancesAndEdlStrengthScratch;
+            }
+        };
+
+        var blendRenderState = RenderState.fromCache({
+            blending : BlendingState.ALPHA_BLEND,
+            depthMask : true,
+            depthTest : {
+                enabled : true
+            }
+        });
+
+        var blendCommand = context.createViewportQuadCommand(blendFS, {
+            uniformMap : blendUniformMap,
+            renderState : blendRenderState,
+            pass : Pass.CESIUM_3D_TILE,
+            owner : processor
+        });
+
+        // set up clear commands for all frame buffers
+        var framebuffers = processor._framebuffers;
+        var clearCommands = {};
+        for (var name in framebuffers) {
+            if (framebuffers.hasOwnProperty(name)) {
+                clearCommands[name] = new ClearCommand({
+                    framebuffer : framebuffers[name],
+                    color : new Color(0.0, 0.0, 0.0, 0.0),
+                    depth : 1.0,
+                    renderState : RenderState.fromCache(),
+                    pass : Pass.CESIUM_3D_TILE,
+                    owner : processor
+                });
+            }
+        }
+
+        processor._drawCommands.blendCommand = blendCommand;
+        processor._clearCommands = clearCommands;
+    }
+
+    function createResources(processor, context, dirty) {
+        var screenWidth = context.drawingBufferWidth;
+        var screenHeight = context.drawingBufferHeight;
+        var colorTexture = processor._colorTexture;
+        var nowDirty = false;
+        var resized = defined(colorTexture) &&
+                      ((colorTexture.width !== screenWidth) ||
+                       (colorTexture.height !== screenHeight));
+
+        if (!defined(colorTexture) || resized || dirty) {
+            destroyFramebuffers(processor);
+            createFramebuffers(processor, context);
+            createCommands(processor, context);
+            nowDirty = true;
+        }
+        return nowDirty;
+    }
+
+    function processingSupported(context) {
+        return context.floatingPointTexture && context.drawBuffers && context.fragmentDepth;
+    }
+
+    PointCloudEyeDomeLighting.processingSupported = processingSupported;
```

Comment:
Rename this to `isSupported` to be consistent with the other files.

Reply:
Since this class is private I think we will also need to also expose this in `PointShading`.